phc/api_response.py

```diff
@@ -26,10 +39,13 @@ def __init__(
         self.status_code = status_code
         self._initial_data = data
         self._client = client
+        if isinstance(data, dict) and data.get("links", {}).get("next"):
```

Comment:
Minor: looks like `data` is expected to be a dictionary every where else. If we are comfortable with that assumption we don't need the `isinstance`, otherwise, a precondition check should be used.